### ISSUE TITLE
Implement nightly builds

### ISSUE THREAD

**Issue by VaneskaSousa** [TotalCross/totalcross#3]
Continuous integration is an important point and we are working to implement it more and more. 

We will start with [nightly builds,](https://blog.testproject.io/2019/10/14/what-are-the-benefits-of-having-nightly-builds/) a feature that consists of automatically generating realizations during the night, avoiding wait by TotalCross users for the release of minor versions to access implemented fixes and features.

Please, feel free to contribute with your questions or suggestions :)

**Comment by acmlira** [comment]
I think we are not working on this scope. Let's rephrase and work on this description better to reopen or create a new issue 😅
